content/md/cartridge-configuration/03-products-filter-configuration.md

```diff
@@ -88,3 +88,18 @@ For example, the filter below enables you to import products with a completeness
     "attributes": ["name", "description", "mytext"]
 }
 ```
+
+# How can I synchronize my product PIM status with the SaloesForce online status (onlineFlag)?
+
+Since the 20.2.2 version, you are able to configure in the [**connector configuration page**](01-where-configuration.html), your choice to use the automated PIM Product synchronization:
```

Comment:
Je mettrais plutôt: "Since the 20.2.2 version, you are able to configure in the [**connector configuration page**](01-where-configuration.html) how you want to synchronize your PIM Product status:"

Reply:
Make sense indeed :)